result/bgpprefix.yaml

```diff
@@ -281,3 +303,340 @@ components:
           type: integer
           format : uint32
           x-field-uid: 9
+    BgpPrefix.RecvExtendedCommunity:
+      description:
+        Each received Extended Community attribute is available for retrieval in two forms.
+        Support of the 'raw' format in which all 8 bytes (16 hex characters) is always present and available for use.
+        In addition, if supported by the implementation, the Extended Community attribute may also be retrieved in the 
+        'structured' format which is an optional field. 
+      type: object
+      properties:
+        raw:
+          description: >-
+            The raw byte contents of the 8 bytes received in the Extended Community as 16 hex characters.              
+          type: string
+          format: hex
+          maxLength: 16
+          x-field-uid: 1  
+        structured:
+          x-field-uid: 2
+          $ref: '#/components/schemas/BgpPrefix.ExtendedCommunity'
+
+    BgpPrefix.ExtendedCommunity:
+      description: >-
+        The Extended Communities Attribute is a optional BGP attribute,defined in RFC4360 with the Type Code 16. 
+        Community and Extended Communities  attributes are utilized to trigger routing decisions, such as acceptance, rejection,  preference, or redistribution. 
+        An extended community is an 8-bytes value. It is divided into two main parts. The first 2 bytes of the community  encode a type and optonal sub-type field.
```

Comment:
pls check the paragraph formatting.

Reply:
These appear to be github review specific views and probably not in our control. Appear to come fine in redocli view.